DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -31,27 +31,21 @@ extern "C" {
 //#pragma GCC diagnostic pop
 
 #ifdef IRODS_HEADER_HPP
-  #include "rodsClient.hpp"
+  #include "irods/rodsClient.hpp"
 #else
-  #include "rodsClient.h"
+  #include "irods/rodsClient.h"
 #endif
 
-#include "irods_query.hpp"
-#include "irods_string_tokenize.hpp"
-#include "irods_virtual_path.hpp"
+#include "irods/irods_query.hpp"
+#include "irods/irods_string_tokenize.hpp"
+#include "irods/irods_virtual_path.hpp"
 #include "irods_hasher_factory.hpp"
-#include "irods_at_scope_exit.hpp"
-#include "get_file_descriptor_info.h"
-#include "replica_close.h"
-#include "thread_pool.hpp"
-#include "filesystem.hpp"
-#ifdef IRODS_HEADER_HPP
-  #include "rodsClient.hpp"
-#else
-  #include "rodsClient.h"
-#endif
-
-#include <base64.h>
+#include "irods/irods_at_scope_exit.hpp"
+#include "irods/get_file_descriptor_info.h"
+#include "irods/replica_close.h"
+#include "irods/thread_pool.hpp"
+#include "irods/filesystem.hpp"
```

Comment:
All of these should be using angle brackets instead of quotation marks, e.g.
```c++
#include <irods/filesystem.hpp>
```
Same for the other iRODS includes throughout this PR.

Reply:
Should we change all of the irods includes or just the ones in this PR?

I generally prefer on this issue to be consistent with the legacy code rather than having a mix of quotes and angle brackets for irods includes but also hesitate to touch legacy code that doesn't require touching.

Reply:
These were all change to angle brackets.